tests/scripts/minikube.sh

```diff
@@ -3,31 +3,24 @@
 scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source "${scriptdir}/../../build/common.sh"
 
-tarname=image.tar
-tarfile=$(pwd)/_output/tests/${tarname}
-
-ssh_opts=(
-  -o StrictHostKeyChecking=no
-  -o "UserKnownHostsFile /dev/null"
-  -o LogLevel=quiet
-)
-
-minikube_scp() {
-    local ip=$(minikube ip)
-    local ssh_key=$(minikube ssh-key)
-    scp "${ssh_opts[@]}" -i ${ssh_key} $1 docker@${ip}:$2
-}
-
-minikube_ssh() {
```

Comment:
why is this minikube_ssh function not needed anymore?

Reply:
To not reinvent the wheel. minikube has a ssh command "minikube ssh".

Reply:
by doing this you've lost the ssh_opts. please revert.

Reply:
i disagree. The ssh_opts were defined because we were implementing our own ssh function for sshing into the minikube vm. Minikube already has a ssh subcommand  that does just that, sshing into the minikube vm.  It is `minikube ssh`. So we shouldnt reinvent the wheel.